open-broker-common/src/main/java/de/fraunhofer/iais/eis/ids/broker/core/common/persistence/SelfDescriptionPersistenceAndIndexing.java

```diff
@@ -82,9 +82,15 @@ public void setContextDocumentUrl(String contextDocumentUrl) {
     public void refreshIndex() {
         //Recreate the index to delete everything
         try {
-            logger.info("Refreshing index.");
+            logger.info("Refreshing indices.");
             indexing.recreateIndex("registrations");
 
+            //If exists, recreate the separate resources index, too
+            try {
+                indexing.recreateIndex("resources");
+            }
+            catch (Exception ignored) {}
+
```

Comment:
Maybe some logging here. Was the index unsuccesful or was there no Index present